doc/man7/provider-signature.pod

```diff
@@ -201,10 +200,22 @@ set of "signature" functions, i.e. at least one of:
 
 =back
 
-OSSL_FUNC_signature_set_ctx_params and OSSL_FUNC_signature_settable_ctx_params are optional,
-but if one of them is present then the other one must also be present. The same
-applies to OSSL_FUNC_signature_get_ctx_params and OSSL_FUNC_signature_gettable_ctx_params, as
-well as the "md_params" functions. The OSSL_FUNC_signature_dupctx function is optional.
+OSSL_FUNC_signature_set_ctx_params and OSSL_FUNC_signature_settable_ctx_params
+function are optional,
```

Comment:
```suggestion
The OSSL_FUNC_signature_set_ctx_params() and OSSL_FUNC_signature_settable_ctx_params()
functions are optional,
```

Reply:
Done, also for all doc improvement suggestions below.